core/icons/comment_icon.ts

```diff
@@ -110,6 +110,7 @@ export class CommentIcon extends Icon implements IHasBubble, ISerializable {
       },
       this.svgRoot,
     );
+    dom.addClass(this.svgRoot, blockly-icon-comment);
```

Comment:
These need to be strings
```suggestion
    dom.addClass(this.svgRoot, 'blockly-icon-comment');
```